src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -276,7 +276,7 @@ private string GetSourceLink(MSBuildItem projectItemInstance)
 			return link;
 		}
 
-		public KeyValuePair<string, string>[] Generate(GenerationRunInfo generationRunInfo)
+		public List<KeyValuePair<string, string>> Generate(GenerationRunInfo generationRunInfo)
```

Comment:
Changed the return type to avoid the unnecessary `.ToArray()` call at the end of this method.